### DISCUSSION TITLE
Validation Error Count?

### DISCUSSION THREAD

**Discussion by TheQuail13** [quasarframework/quasar#10674] [General - Components / Directives / etc] [answered]
After validating a list of fields that are marked as required (using the internal quasar field validation), is it possible to find a count of how many errors exist within a component? For my current use-case, it would be awesome to show to the user how many errors exist for a given step in their workflow, I'm just not sure how to go about arriving at that count. 

Or as an alternative, is there an existing way to see if any errors exist within a component? My end-goal is to display if there are any errors present within a q-expansion-item (that contains the input fields) which may be collapsed when they run the validation.

Thank you.

**Comment by pikil** [comment, marked as answer]
@TheQuail13 If you know your inputs and can access them by ref, you can use ```hasError``` property of that element to count how many have ```true``` - that would be the count you need.
